mappings/net/minecraft/resource/ReloadableResourceManagerImpl.mapping

```diff
@@ -5,5 +5,12 @@ CLASS xi net/minecraft/resource/ReloadableResourceManagerImpl
 	FIELD d initialListeners Ljava/util/List;
 	FIELD e namespaces Ljava/util/Set;
 	FIELD f type Lwi;
+	FIELD g engineThread Ljava/lang/Thread;
```

Comment:
`engineThread`? Could just `thread` be enough (as in, "the thread of the resource manager").

Reply:
It's the thread of the engine (client, server threads). Not used by the manager.

Reply:
Ah. I think it's called `mainThread` in other places (chunk loading code, and also the thread names), so it should probably be called that to be consistent.